js/food.js

```diff
@@ -0,0 +1,62 @@
+// XMLHttpRequests are REALLY HARD and I hope I did this in a sustainable manner
+
+// Elements for inserting information into the Dog Food Table in the DOM
+var products = document.getElementById('products');
+var dogBrands = document.getElementsByClassName('brand-dog');
+var dogType = document.getElementsByClassName('types-dog');
+
+// Nesting ALL the for-loops to get the objects to appear in the order I want in the DOM
+// STILL SURPRISED THAT THIS WORKS, WHAT KIND OF WITCHCRAFT IS THIS?!
```

Comment:
It's not witchcraft so much as the JSON is formatted in such a way that the nesting of it matches the nesting you want on the page.  Generally this will be the case, as formatting the data in the JSON in a logical manner is usually the same logical manner you'd want to display them on the page. :wink: